ballerina-tests/tests/45_services_with_headers_service.bal

```diff
@@ -54,23 +54,23 @@ service "HeadersService" on ep45 {
             {name: "Miller", message: "Bro"}
         ];
         foreach HSRes res in responses {
-            check caller->sendContextHSRes({content: res, headers: responseHeaders});
+            checkpanic caller->sendContextHSRes({content: res, headers: responseHeaders});
         }
-        check caller->complete();
+        checkpanic caller->complete();
     }
 
-    remote function clientStr(HeadersServiceHSResCaller caller, ContextHSReqStream req) returns error? {
+    remote function clientStr(HeadersServiceHSResCaller caller, ContextHSReqStream req) {
         map<string|string[]> responseHeaders = {};
         if !req.headers.hasKey("client-steaming-req-header") {
-            check caller->sendError(error grpc:AbortedError("client-steaming-req-header header is missing"));
+            checkpanic caller->sendError(error grpc:AbortedError("client-steaming-req-header header is missing"));
             return;
         } else {
-            string headerValue = check grpc:getHeader(req.headers, "client-steaming-req-header");
+            string headerValue = checkpanic grpc:getHeader(req.headers, "client-steaming-req-header");
```

Comment:
```suggestion
            string headerValue = check grpc:getHeader(req.headers, "client-steaming-req-header");
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/2ef479d6607ef5aa5336331221c12c698cdd94cb